src/core/server/http/router/request.ts

```diff
@@ -97,6 +97,7 @@ export class KibanaRequest<Params = unknown, Query = unknown, Body = unknown> {
 
   public readonly url: Url;
   public readonly route: RecursiveReadonly<KibanaRequestRoute>;
+  public readonly headers: Headers;
```

Comment:
Can we add a TSDoc comment here calling this out as the non-preferred way of doing this?